db/app.js

```diff
@@ -1,18 +1,39 @@
 const express = require("express");
-const { getTopics } = require("./controllers/topics.controller.js");
+const { getTopics, getArticle } = require("./controllers/topics.controller.js");
 const app = express();
+const endpoints = require("../endpoints.json");
 
 app.use(express.json());
```

Comment:
We don't actually need this line of code until we make a post request. Be careful to only include logic to fulfil the ticket in hand